src/middlewared/middlewared/plugins/jail.py

```diff
@@ -37,6 +38,10 @@
 SHUTDOWN_LOCK = asyncio.Lock()
 
 
+class OfficialPluginRepositories(enum.Enum):
+    IXSYSTEMS = 'ixSystems'
```

Comment:
iXsystems is what we call it

Reply:
Updating asap

Reply:
Also not sure how an ENUM helps here?

I mentioned enum in the PR just as an acronym (C lang), not actual enum python module